parsl/dataflow/futures.py

```diff
@@ -131,15 +126,15 @@ def parent_callback(self, executor_fu):
 
     @property
     def stdout(self):
-        return self._stdout
+        return self.task_def['kwargs']['stdout']
```

Comment:
I think it is ok for these to be undefined, so this should return `self.task_def['kwargs'].get('stdout')` (and similarly for stderr below).